.github/workflows/publish.yml

```diff
@@ -61,10 +61,18 @@ jobs:
       - run: mkdir reports
       - run: yarn test:ci
 
-  publish_build_image_to_registries:
+  run_vulnerability_scan_on_repo:
     needs:
       - integrated_tests
       - get_variables
+    uses: telicent-oss/shared-workflows/.github/workflows/vulnerability-scanning-on-repo.yml@TELFE-187
```

Comment:
**MAY BE IRRELEVANT**
Note: On my shared-workflows which might be old...
I had to add `secrets: inherit`,[but it still failed](https://github.com/telicent-oss/telicent-instance/actions/runs/9194592999/job/25288742612#step:5:23)